### PR TITLE
chore: workflow changes to accomodate the new release branches

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: .github/workflows/prbuild.yml [hunk 1]
```diff
@@ -17,5 +19,13 @@ jobs:
...
       - run: make test-coverage
-      - name: upload coverage
+      - id: branch_info
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const resp = await github.request('GET /repos/{owner}/{repo}/pulls/{pull_number}', {...context.repo, pull_number:context.payload.number});
+            core.setOutput('branch', resp.data.base.ref);
+      - if: ${{ steps.branch_info.outputs.branch == 'main' }}
```
Comment (ashokponkumar) [inline .github/workflows/prbuild.yml:29]: Why not upload coverage for all branches?

**Review by ashokponkumar — CHANGES_REQUESTED** [review]

File: .github/workflows/prbuild.yml (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply .github/workflows/prbuild.yml:29]: Not sure if that's how it works.
Doesn't it just overwrite the coverage of the main branch?

Comment (ashokponkumar) [reply .github/workflows/prbuild.yml:29]: I don't think so. The ui seems to have branches and stuff. Even if not the action would take as input.

Comment (HarikrishnanBalagopal) [reply .github/workflows/prbuild.yml]: done

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**